yb-voyager/cmd/live_migration.go

```diff
@@ -243,15 +244,24 @@ func processEvents(chanNo int, evChan chan *tgtdb.Event, lastAppliedVsn int64, d
 	done <- true
 }
 
-func updateExportedEventsStats(statsReporter *reporter.StreamImportStatsReporter) {
+func updateExportedEventsStats() {
 	ticker := time.NewTicker(10 * time.Second)
 	defer ticker.Stop()
 
 	for range ticker.C {
-		totalExportedEvents, _, err := metaDB.GetTotalExportedEvents(time.Now().String())
-		if err != nil {
-			utils.ErrExit("failed to fetch exported events stats from meta db: %v", err)
-		}
-		statsReporter.UpdateRemainingEvents(totalExportedEvents)
+		fetchAndUpdateEventsStats()
+	}
+}
+
+func fetchAndUpdateEventsStats() {
+	totalExportedEvents, _, err := metaDB.GetTotalExportedEvents(time.Now().String())
```

Comment:
how about having a retry logic, maybe 5 times before exiting the process here?

Reply:
How will that help? like in which case retry can succeed?

Reply:
I am not 100% sure, but there is one error we throw based on this condition:
```
if !strings.Contains(err.Error(), "converting NULL to int64 is unsupported")
```
when will that occur?

can this error occur in case the stats table is not populated by the debezium at initial moments?

Reply:
this error is handled inside the `GetTotalExportedEvents()` function in `metadataDB.go`. I don't it will occur here.

Reply:
I know that. I am asking when will db throw that error?
Maybe can this error occur in case the stats table is not populated by the debezium at initial moments?